src/css/components/_control-bar.scss

```diff
@@ -10,6 +10,15 @@
   @include background-color-with-alpha($primary-background-color, $primary-background-transparency);
 }
 
+// IE8 is flakey with fonts, and you have to change the actual content to force
```

Comment:
oh, I see, it's for the variable. Maybe move just the variable but lease the section where it was?